chainer/optimizers/adam.py

```diff
@@ -78,14 +89,20 @@ class AdamRule(optimizer.UpdateRule):
         eta (float): Schedule multiplier, can be used for warm restarts.
         weight_decay_rate (float): Weight decay rate.
         amsgrad (bool): Whether to use the AMSGrad variant of Adam.
+        adabound (bool): Whether to use the AdaBound variant of Adam.
+        final_lr (float): final (SGD) learning rate in adabound.
```

Comment:
nit-picking: How about beginning the explanation of `final_lr` with a capital letter for consistency with the other arguments?
```suggestion
        final_lr (float): Final (SGD) learning rate in AdaBound.
```